src/js/cypress/Browser.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable class-methods-use-this */
 import "@cypress/xpath";
 
-export type BrowserTimeout = "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl" | "xxxl";
+export type BrowserTimeout = "zero" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl" | "xxl" | "xxxl";
```

Comment:
Nit, but we could make these into enums to avoid having to remember the strings

TimeoutEnums.ExtraExtraSmall
TimeoutEnums.ExtraSmall
TimeoutEnums.Small

etc.

Reply:
YES! But ExtraShort, ExtraLong etc -- it's temporal measurement. Or I'd prefer something that resembles actual duration, often you know exactly, that the response will be in 45-50 seconds, why you should guess whether it's Medium, Long, Short?